modules/openapi-generator/src/main/resources/avro-schema/dataType.mustache

```diff
@@ -0,0 +1 @@
+{{#useLogicalTypes}}{{#isDate}}{ "type": "int", "logicalType": "date" }{{/isDate}}{{#isDateTime}}{ "type": "long", "logicalType": "timestamp-{{{logicalTypeTimeQuantifier}}}" }{{/isDateTime}}{{^isDate}}{{^isDateTime}}"{{{dataType}}}"{{/isDateTime}}{{/isDate}}{{/useLogicalTypes}}{{^useLogicalTypes}}"{{{dataType}}}"{{/useLogicalTypes}}
```

Comment:
would be great to format this better; if there is a way to use whitespace in the template and then make it collapse in the result, please let me know.

Reply:
are you looking for something like following?

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/Javascript/libraries/javascript/api.mustache#L3
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/Javascript/libraries/javascript/api.mustache#L118

Reply:
no, that's changing the template operator, right? I basically wish I could change the string in this template to:
```suggestion
{{#useLogicalTypes}}
  {{#isDate}}
    {
      "type": "int",
      "logicalType": "date"
    }
  {{/isDate}}
  {{#isDateTime}}
    {
     "type": "long",
     "logicalType": "timestamp-{{{logicalTypeTimeQuantifier}}}"
    }
  {{/isDateTime}}
  {{^isDate}}
    {{^isDateTime}}
      "{{{dataType}}}"
    {{/isDateTime}}
  {{/isDate}}
{{/useLogicalTypes}}
{{^useLogicalTypes}}
  "{{{dataType}}}"
{{/useLogicalTypes}}
```
and still have well-spaced output in the resulting schema. Something like a `{{#collapseWhitespace}}...{{/collapseWhitespace}}` operator that takes anything in its body and reduces all `\w+` to ` `

Reply:
nope. i don't think mustache comes with anything to do that

looks like avsc file are just JSON file so users can use a linter to auto format the auto-generated output as they do with other generators